### DISCUSSION TITLE
微调的时候 数据必须是一问一答的形式吗 还是也可以多问多答的

### DISCUSSION THREAD

**Discussion by zhaoyz88322** [QwenLM/Qwen#440] [Q&A]
微调的时候 数据必须是一问一答的形式吗？
[
  {
    "id": "identity_0",
    "conversations": [
      {
        "from": "user",
        "value": "你好"
      },
      {
        "from": "assistant",
        "value": "我是一个语言模型，我叫通义千问。"
      }
    ]
  }
]
还是也可以像这样 多问多答的
[
  {
    "id": "identity_0",
    "conversations": [
      {
        "from": "user",
        "value": "你好"
      },
      {
        "from": "assistant",
        "value": "我是一个语言模型，我叫通义千问。"
      },
{
        "from": "user",
        "value": "编写一首诗"
      },
      {
        "from": "assistant",
        "value": "好的，山下一群鹅，嘘声赶下河"
      }
    ]
  }
]

**Comment by jklj077** [comment]
我理解这个例子里“多问多答”指多轮的一问一答，这个可以的，很多开源模型都是这么样训练的，比如vicuna之类的。
